### PR TITLE
release code optimisation

### PR DESCRIPTION
**Pull request by ketanshikharebsft**
Added set object method for setting timestamp in dictionary
Replaced ceil with round for % to points conversion

### CODE REVIEW

File: BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m [hunk 1]
```diff
@@ -72,13 +72,13 @@ + (CGFloat)convertPointsWidthToPercentage:(float) width forWindow:(UIWindow*)win
 
 + (CGFloat)convertPercentageHeightToPoints:(float) height forWindow:(UIWindow*)window {
     CGFloat presentationAreaHeight = [self getPresentationAreaHeightForWindow:window];
-    CGFloat heightInPoints = (CGFloat) ceil(presentationAreaHeight * (height / 100.0f));
+    CGFloat heightInPoints = (CGFloat) round(presentationAreaHeight * (height / 100.0f));
```
Comment (rahulrvp) [inline BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m:75]: What is the difference between these two methods?

Comment (ketanshikharebsft) [reply BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m:75]: convertPercentageHeightToPoints is converting the given in-app percentage height to points height and
convertPercentageWidthToPoints is converting the given in-app percentage width to points width

Comment (rahulrvp) [reply BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m:75]: No, I was asking about the ceil and round functions. :) I guess ceil rounds everything up to the next decimal where round only rounds if the decimal value is .5 or above.

**Review by rahulrvp — APPROVED** [review]
LGTM

File: BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m (hunk 1 above)
Comment (ketanshikharebsft) [reply BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m:75]: Yes, that's correct.

**Pull request merged**